packages/devtools/src/index.ts

```diff
@@ -0,0 +1,44 @@
+import devtools from '@vue/devtools'
+import express from 'express'
+// @ts-ignore
+import launchMiddleware from 'launch-editor-middleware'
+
+export interface DevtoolsOptions {
+  title?: string
+  host?: string
+  openInEditor?: boolean
+}
+
+const SERVER_PORT = 8098
+
+export function createDevtools(options: DevtoolsOptions = {}) {
+  const {
+    host = 'http://localhost',
+    openInEditor = true,
+    title = 'vue-termui devtools',
+  } = options
+
+  // workaround for @vue/devtools
```

Comment:
This should be fixable in @vue/devtools as it has been done in the past like https://github.com/vuejs/devtools/pull/1780/files